packages/core/src/utils/node.ts

```diff
@@ -0,0 +1,12 @@
+import cheerio from 'cheerio';
+import { DomElement } from 'htmlparser2';
+
+export type TextElement = DomElement;
+
+export type Node = DomElement & cheerio.Element & {
```

Comment:
As @raysonkoh suggested, maybe we should rename this to something like `NodeElement` or `MarkbindNode`.

Reply:
I'd suggest `NodeElement`, wdyt?

Reply:
I think `NodeElement` might be a little wordy. How about `MbNode`?

Reply:
👍  renamed to `MbNode`